src/graphql/root/query/wallet-name-available.ts

```diff
@@ -0,0 +1,29 @@
+import { walletNameAvailable } from "@app/wallets/wallet-name-available"
```

Comment:
Here we can also follow the pattern of `import * as Wallets` as in the other files I think.